mkdocs.yml

```diff
@@ -0,0 +1,62 @@
+site_name: "Office-of-Digital-Services/cdt-ods-disaster-recovery: Documentation"
+repo_url: https://github.com/Office-of-Digital-Services/cdt-ods-disaster-recovery
+edit_uri: edit/main/docs
+site_url: https://Office-of-Digital-Services.github.io/cdt-ods-disaster-recovery
+
+theme:
+  name: material
+  features:
+    - navigation.expand
+    - navigation.tabs
+    - toc.integrate
+  palette:
+    scheme: default
+
+extra:
+  analytics:
+    provider: google
+    property: G-xxxxxxxxxx
```

Comment:
Should we omit this section for now if we don't have Google Analytics for this site?

Reply:
Yep, that makes sense! I removed it in 23557c841fcda5d79d6a56db589502c51eb5dbf6